### DISCUSSION TITLE
Allow the ability to hide/lock client permissions via Strategy

### DISCUSSION THREAD

**Discussion by andrewheberle** [rustdesk/rustdesk-server-pro#279] [Ideas]
In a corporate environment, allowing the remote user to control the permissions of the session is a problem and should be controllable via a Strategy.

Ideally this part of the UI would be completely hidden if all permissions were locked in the Strategy.

I had originally added this as a discussion item in the rustdesk/rustdesk repo but this is probably a more appropriate location.

**Comment by ChaosNicro** [comment]
Yes, maybe the relevant advanced/override options could also have a setting to hide the toggle during a session.
Isn't this more a feature request, than an idea?

**Comment by ChaosNicro** [comment]
"enable-perm-change-in-accept-window=N" combined with override settings for the individual permissions achieve this effect. 
The ability to outright hide the permissions pane in the accept-window would still be great, if only to provide a clean UX for the user. 
Showing active permissions  to the user might be required for compliance in some contexts, but not in all cases.
